### DISCUSSION TITLE
Prusa slicer importing SL1 files

### DISCUSSION THREAD

**Discussion by eddy-99** [sn4k3/UVtools#314] [Q&A]
I am trying to get a workflow going.
I am using **PrusaSlicer-2.4.0-alpha3+win64-202109301506**
STL into prusa save as SL1 fix pixel problems with UVTools and save as SL1 and import back into prusa but I get
error **Invalid value provided for parameter host_type: ** when importing back into prusa.
I have tried SL1s as well with the same problem.

Also is there a way to import more than one object at a time into UVTools so they could be edited and saved?

Thanks Eddy

**Comment by sn4k3** [comment]
I only support the stable releases, but i can have a look into that

**Reply by eddy-99** [reply]
I had the same problem with Prusa 2.3.3

**Reply by sn4k3** [reply]
Ok, i had the 2.3.1 installed, i will update and test that

**Reply by sn4k3** [reply]
It works in current version for me:

![prusa-slicer_2021-10-07_16-16-01](https://user-images.githubusercontent.com/113281/136413765-e4ab6032-15e4-4751-9c0c-5e3e5bc0f643.png)

Did you use UVtools to convert file? (File - Convert?)

**Reply by eddy-99** [reply]
STL into prusa save as sl1 open the sl1 in UVTools once edited convert to sl1 but cannot get it back into prusa

**Reply by sn4k3** [reply]
> ... in UVTools once edited convert to sl1 ....

Why are you converting from sl1 to sl1? All you need is use File - Save. Convert to SL1 will make you loss configurations

**Reply by eddy-99** [reply]
Thanks my mistake again !
One more question, I import a figure which has a cavity, I solidify and check layers and any missing pixels I paint in.
When saved as sl1 and back into prusa the cavity is still there
![Capture](https://user-images.githubusercontent.com/57560700/136418647-9ea8d998-7089-4255-bc20-475082f9a24c.PNG)
?

**Reply by sn4k3** [reply]
Need to check that, send me the file and tell me which layer index is the cavity.
I solidified a model, and it work when import back with holes filled

**Reply by eddy-99** [reply]
Layer 576 to 598 mailed the file

**Comment by eddy-99** [comment]
[005 Standing Foot On Crate Apron Casual  01.zip](https://github.com/sn4k3/UVtools/files/7305267/005.Standing.Foot.On.Crate.Apron.Casual.01.zip)

**Reply by sn4k3** [reply]
You selected Longer Orange 10 on prusa slicer.
UVtools auto converted .SL1 to .LGS once open the .sl1, so you are editing .lgs and the .sl1 will be kept as original file unchanged... 
Thats why you see no change when you open the .sl1. 
I assume you have a longer orange? 
In this case you need to go to UVtools settings -> Automations and disable the auto convertion. And in the end you must manual convert to .lgs when you ready to make the transfer

**Comment by eddy-99** [comment]
Getting there thanks for your help.
just get this in Prusa now!
![Capture](https://user-images.githubusercontent.com/57560700/136428487-8be0550a-d0a2-4ce7-8de4-4f33d59ce39b.PNG)
!

**Reply by sn4k3** [reply]
Ignore those warnings, i will try to fix that on next patch
